src/pages/SignUp/SignUp.tsx

```diff
@@ -96,6 +100,7 @@ const SignUp = () => {
         </AuthForm>
         <AuthSub>Already have an account?</AuthSub>
         <AuthRedirectLink to="/sign-in">Login Here</AuthRedirectLink>
+        <CookieRequest>Please make sure cookies are enabled.</CookieRequest>
```

Comment:
<img width="768" alt="alert" src="https://user-images.githubusercontent.com/36215621/172920173-84d89e01-87ec-43e0-8517-fff93e4365ca.png">

What do you think if we have something like this on top of the sign in card?

Reply:
I like that! Should it be a static alert that is always visible? 

I can do something like this using one of the colors from our theme file.